vllm/model_executor/models/bart.py

```diff
@@ -762,6 +762,74 @@ def forward(self, input_ids: torch.Tensor, positions: torch.Tensor,
 
         return decoder_outputs
 
+    def load_weights(self, weights: Iterable[tuple[str, torch.Tensor]]):
+        model_params_dict = dict(self.model.named_parameters())
```

Comment:
This should is `self.model.named_parameters()`, because `self.model` is in `BartForConditionalGeneration` call.

Reply:
updated